examples/plugin/settings.gradle

```diff
@@ -0,0 +1,8 @@
+rootProject.name = 'plugin-example'
+
+// There are no published ORT binaries that this example could depend on, so use a Gradle composite build.
+includeBuild('../..') {
+    dependencySubstitution {
+        substitute module('com.here.ort:cli') with project(':cli')
```

Comment:
You can now even add a rationale via `because`, see https://docs.gradle.org/current/userguide/customizing_dependency_resolution_behavior.html#example_substituting_a_module_with_a_project, to get rid of the code comment above.